src/families/ethereum/synchronisation.ts

```diff
@@ -362,6 +523,7 @@ const txToOps =
         extra: {},
         internalOperations,
         subOperations,
+        nftOperations,
```

Comment:
if these are considered as contract operations shouldnt they be put/merged inside the subOperations maybe :thinking: 
it doesnt make much sense to me to separate the two

Reply:
As discussed already with you on slack, I think we need that in order to be non breaking since subOps and nftOps are different types. But I really don't enough about the usage in LLD/LLM so it might be _safe—ish_ to concat them all. I need to dig a little bit more into that 🤷‍♂️

Reply:
no, subOperations are strictly for "operations of sub accounts". I believe here is the correct choice.